### DISCUSSION TITLE
Keyboard problem with Chrome + Ubuntu

### DISCUSSION THREAD

**Discussion by bero** [gravitational/teleport#40073] [Q&A]
I am a member in a team that have Windows in GCP (Google Cloud Platform) and I use Teleport.
I use Ubunt0 23.10 as OS and KVM 8.0.4 as virtualization.
GCP is Windows Server 2022 Datacenter version 21H2.

Generally it works well but there are some problems. To get discussion more focused I only write one problem in each thread.

- I have also tried login with Chrome on Ubuntu. It has same login issue as Windows but in addition to that left Ctrl key is not active in any application. This is serious as hotkeys is commonly used. Ctrl+Z become same as type z. Ctrl + F same as type f etc. So I must login from Windows guest to make this work. I have verified this on sites like https://www.keyboardtester.com and https://en.key-test.ru

**Comment by webvictim** [comment]
I've created a bug report to track this: https://github.com/gravitational/teleport/issues/40207

**Comment by danielashare** [comment]
Hi @bero, are you still experiencing this issue? I'm struggling to reproduce it.
